google-ads-migration-examples/pom.xml

```diff
@@ -0,0 +1,105 @@
+<!--
```

Comment:
This copyright is not the standard one used on `.java` and other source files.

@nwbirnie I see that we have this copyright in the existing `pom.xml` files as well. Do you recall why, or did I do that? :smile:

Reply:
Done (will change other files in a subsequent PR).